.github/workflows/make-test.yml

```diff
@@ -31,12 +31,6 @@ jobs:
           brew install llvm@13
           echo "/usr/local/opt/llvm@13/bin" >> $GITHUB_PATH
           echo "LLVM_SUFFIX=" >> $GITHUB_ENV
-      - name: Install LLVM (ubuntu)
```

Comment:
@jclark it turned out github's version of ubuntu 22.04 already has llvm 13. So I removed the install step